### PR TITLE
Use fast gas price strategy to calculate gas price

### PR DESCRIPTION
**Pull request by fredo**
Although the gas price strategy is set to fast, it was totally ignored in the whole project by using the direct RPC `web3.eth.gasPrice`. Using `web3.eth.generateGasPrice()` instead calculates the gas price based on the strategy.

### CODE REVIEW

File: raiden_installer/web.py [hunk 1]
```diff
@@ -329,7 +337,7 @@ def _run_track_transaction(self, **kw):
         configuration_file_name = kw.get("configuration_file_name")
         tx_hash = kw.get("tx_hash")
         time_elapsed = 0
-
+        time.sleep(30)
```
Comment (konradkonrad) [inline raiden_installer/web.py:340]: is this sleep intentional?

Comment (fredo) [reply raiden_installer/web.py:340]: how did that slip through?

Comment (fredo) [reply raiden_installer/web.py]: We have a problem though that track transaction is called before the node has received the tx_hash. There is already a 30s sleep in place

**Comment by agatsoh** [comment]
LGTM

**Pull request merged**